ang/civicase/contact-case-tab/directives/contact-case-tab.directive.js

```diff
@@ -38,6 +38,7 @@
     };
 
     $scope.caseDetailsLoaded = false;
+    $scope.caseTypeCategoryName = CRM['civicase-base'].caseTypeCategories[$scope.caseTypeCategory].name;
```

Comment:
Cant we access case category name directly from the url? Using `parseUrlParameters` factory.

Also if we cant, we should not use `CRM['civicase-base'].caseTypeCategories` directly. Should use `CaseTypeCategory.getAll` service.